scripts/prepare-testing-project.sh

```diff
@@ -44,7 +44,7 @@ echo "Creating pubsub resources."
 gcloud alpha pubsub topics create gae-mvm-pubsub-topic
 
 echo "Creating speech resources."
-gsutil cp speech/api/resources/audio.flac gs://$GCLOUD_PROJECT/speech/
+gsutil cp speech/api-client/resources/audio.raw gs://$GCLOUD_PROJECT/speech/
```

Comment:
Just searched through the repo and it doesn't look like there are. Also, the audio.flac file doesn't exist in that directory anymore.